src/angular-youtube-embed.js

```diff
@@ -1,3 +1,7 @@
+if (!angular && require){
+    var angular = require('angular');
+}
```

Comment:
This is the correct way to make it work in browserify and webpack. Otherwise it needs to be injected into teh namespace, which is ugly and requires extra configuring.